test/integration/smart_answer_flows/calculate_your_holiday_entitlement_test.rb

```diff
@@ -0,0 +1,377 @@
+require_relative "../../test_helper"
+require_relative "flow_test_helper"
+
+require "smart_answer_flows/calculate-your-holiday-entitlement"
+
+class CalculateYourHolidayEntitlementTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup do
+    setup_for_testing_flow SmartAnswer::CalculateYourHolidayEntitlementFlow
+  end
+
+  context "start page" do
+    should see_start_page("calculate-your-holiday-entitlement").with_title("Calculate holiday entitlement")
```

Comment:
I'm assuming we don't actually need to name the flow given we're in it.

```suggestion
    should see_start_page.with_title("Calculate holiday entitlement")
```

Reply:
Yup, totally agree.  Updated.